enterprise_access/apps/subsidy_request/tasks.py

```diff
@@ -27,3 +33,87 @@ def update_course_title_for_subsidy_request_task(subsidy_type, subsidy_request_u
 
     # Use bulk_update so we don't trigger save() again
     subsidy_model.bulk_update([subsidy_request], ['course_title'])
+
+
+def _get_manage_requests_url(subsidy_model, enterprise_slug):
+    """
+    Get a manage_requests url based on the type of subsidy.
+
+    Args:
+        subsidy_model (class):  class of the subsidy object
+        enterprise_slug (string): slug of the enterprise's name
+    Returns:
+        string: a url to the manage learners page.
+    """
+    if subsidy_model == apps.get_model('subsidy_request.LicenseRequest'):
+        subsidy_string = 'subscriptions'
+    else:
+        subsidy_string = 'coupons'
+
+    url = f'{settings.ENTERPRISE_ADMIN_PORTAL_URL}/{enterprise_slug}/admin/{subsidy_string}/manage-requests'
+    return url
+
+
+@shared_task(base=LoggedTaskWithRetry)
+def send_admins_email_with_new_requests_task(enterprise_customer_uuid):
+    """
+    Task to send new-request emails to admins.
+
+    Args:
+        enterprise_customer_uuid (str): enterprise customer uuid identifier
+    """
+    braze_client = BrazeApiClient()
+    lms_client = LmsApiClient()
+    enterprise_customer_data = lms_client.get_enterprise_customer_data(enterprise_customer_uuid)
+
+    contact_email = enterprise_customer_data.get('contact_email')
```

Comment:
clarification: Is this the admin email/the email we want to use? Most of the contact email is null for the enterprises (only 23 have it set). Or should we fetch enterprise admins and send emails to those?

Reply:
I honestly thought this was the field we were using in other places :/ 

what do you mean by "fetch enterprise admins" in this case?

Reply:
we can query the lms for the enterprise admins, there is an example in license-manager.

Reply:
updated in latest commit